README.md

```diff
@@ -18,31 +16,32 @@ To run the services locally, run the following in your Windows command prompt:
 # create a virtual environment
 python -m venv env
 # active the virtual environment
-. env/Scripts/activate
+.\env\Scripts\activate
 # install the project's dependencies
 pip install -r requirements.txt
 # deploy at a local server
-uvicorn Main:app --host 0.0.0.0 --port 8000
+uvicorn main:app --host 0.0.0.0 --port 8000 --reload
```

Comment:
Should we get rid of reload?

Reply:
I think it depends. For development, it's very useful, but you absolutely do not want it for a server deployment. Maybe mention both scenarios in the README?